src/tlo/methods/healthsystem.py

```diff
@@ -1277,10 +1277,16 @@ def schedule_hsi_event(
         """
         # If there is no specified tclose time then set this to a week after topen.
         # This should be a boolean, not int! Still struggling to get a boolean variable from resource file
-        if self.tclose_overwrite == 1:
-            tclose = topen + pd.to_timedelta(self.tclose_days_offset_overwrite, unit='D')
-        elif tclose is None:
-            tclose = topen + DateOffset(days=7)
+
+        # Clinical time-constraints are embedded in tclose for these modules, do not overwrite their tclose
+        if hsi_event.module == ('PregnancySupervisor' or 'Labour' or 'PostnatalSupervisor' or 'NewbornOutcomes'):
```

Comment:
do we not want `hsi_event.module == ('CareOfWomenDuringLabour' or 'Labour' or 'PostnatalSupervisor' or 'NewbornOutcomes')` instead?

Reply:
*CareOfWomenDuringPregnancy

Reply:
Yes sorry, have fixed now.